c++/src/RLE.hh

```diff
@@ -68,7 +76,24 @@ namespace orc {
      * record current position
      * @param recorder use the recorder to record current positions
      */
-    virtual void recordPosition(PositionRecorder* recorder) const = 0;
+    virtual void recordPosition(PositionRecorder* recorder) const;
+
+  protected:
+    std::unique_ptr<BufferedOutputStream> outputStream;
+    size_t bufferPosition;
+    size_t bufferLength;
+    size_t numLiterals;
+    int64_t* literals;
+    bool isSigned;
+    char* buffer;
```

Comment:
Is this initialized anywhere?

Reply:
Added initialisation to constructor to initialised it to nullptr.